### ISSUE TITLE
setpriv support

### ISSUE THREAD

**Issue by agnostic-apollo** [termux/termux-packages#4936]
Are their any plans for `setpriv` from `util-linux` being included in official termux repo anytime in future? or are their compatibility issues? somebody added a patch for it long ago [here](https://github.com/termux/termux-packages/blob/156a2db7c654a0d7d4d25fae65e8e82044440a1f/packages/util-linux/fix-paths.patch#L404), not sure if others are required.

I was planning to run automated `apt` commands with termux context from tasker with root using [this](https://android.stackexchange.com/questions/217016/how-to-run-a-program-in-an-app-context-with-magisk/217104#217104) and only `setpriv` is missing. I tested the aarch64 binary provided in the link and everything worked including `apt`. But the binary is likely compiled with `musl-libc`.

That's the only binary missing from termux to get everything running, if there are no official plans, I'll have to host it on my own repo.

Thanks

**Comment by RossComputerGuy** [comment]
`setpriv` uses a system call which requires root

**Comment by agnostic-apollo** [comment]
Yeah I'm aware of that. I posted it here because `util-linux` is provided through here. I'm sure some other binaries provided through here also require root, like `busybox mount`...

**Comment by RossComputerGuy** [comment]
But `util-linux` is different, it provides things like kill, more, ls, rev, cal, etc. But root requring packages are in the termux-root repo, not this repo. I'd say it won't be added but it could be added to termux-root by configuring `util-linux` to build only `setpriv`.

**Comment by agnostic-apollo** [comment]
`busybox` also provides a lot of non-root applets and some root ones, although not all are [symlinked](https://github.com/termux/termux-packages/commit/fbd388f695eeacaf5c3895d8f91953ba218664b9).

Anyways, if the maintainers think that this should be moved to the root repo, I'm fine with that. Just wanted to know if there are any plans. I'll be hosting my own repo soon, so I can compile `setpriv` myself as I will be providing a few other root binaries as well, like [fusermount](https://github.com/agnostic-apollo/fuse) and [bindfs](https://github.com/agnostic-apollo/bindfs) that are not provided by termux repos. Conflicts in future is one of the concerns.
